l2tdevtools/dependency_writers/pipenv.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
This should be pipfile.py. It looks like the pipfile is planned to replace requirements.txt, and pipenv is just one tool that will read this format: https://github.com/pypa/pipfile

Reply:
Docstring says `Writer for pipenv files.` the rename is just your personal preference there is nothing that that says this "should" be named as such. Please either provide argumentation why this "should" be renamed or just state that this is your personal preference and why.

Reply:
> Docstring says `Writer for pipenv files.` The rename is just your personal preference there is nothing that that says this "should" be named as such. Please either provide objective argumentation why this "should" be renamed or just state that this is your personal preference and why.

The other dependency writers are (mostly) named after the thing they write - travis_yml.py writes a travis.yml file, pylint_rc.py writes a pylintrc file, requirements writes a requirements.txt file. These files aren't named "travis" (the travis writer actually generates a bash script), "pylint" and "pip" for the tools that read the output. For consistency, this file should also be named after the thing it generates, not the program the consumes the output of the thing it generates.

> 
> > It looks like the pipfile is planned to replace requirements.txt
> 
> I opt to keep it as-is for now and change it when this is planned replacement is reality

I agree, I wasn't suggesting this. I meant that other programs that pipenv will likely read this file.